src/Sylius/Behat/Context/Api/Shop/CustomerContext.php

```diff
@@ -246,6 +296,50 @@ public function iShouldBeNotifiedThatFirstNameIsRequired(): void
         ));
     }
 
+    /**
+     * @Then I should be notified that the first name needs to be provided
+     */
+    public function iShouldBeNotifiedThatFirstNameNeedsToBeProvided(): void
+    {
+        Assert::true($this->isViolationWithMessageInResponse(
+            $this->client->getLastResponse(),
+            'Please enter your first name.',
+        ));
+    }
+
+    /**
+     * @Then I should be notified that my gender is invalid
+     */
+    public function iShouldBeNotifiedThatGenderIsInvalid(): void
+    {
+        Assert::true($this->isViolationWithMessageInResponse(
+            $this->client->getLastResponse(),
+            'The value you selected is not a valid choice.',
+        ));
+    }
+
+    /**
+     * @Then I should be notified that the phone number is too long
+     */
+    public function iShouldBeNotifiedThatThePhoneNumberIsTooLong(): void
+    {
+        Assert::true($this->isViolationWithMessageInResponse(
+            $this->client->getLastResponse(),
+            'Phone number must not be longer than 255 characters.',
+        ));
+    }
+
+    /**
+     * @Then I should be notified that the last name needs to be provided
+     */
+    public function iShouldBeNotifiedThatLastNameNeedsToBeProvided(): void
+    {
+        Assert::true($this->isViolationWithMessageInResponse(
+            $this->client->getLastResponse(),
+            'Please enter your last name.',
+        ));
+    }
```

Comment:
You can merge this step with the `iShouldBeNotifiedThatFirstNameNeedsToBeProvided`